features/verblijfstitelhistorie.feature

```diff
@@ -31,8 +31,8 @@
 
   Als een verblijfstitel, actueel of historisch, in onderzoek is, en dit onderzoek is niet afgerond (Datum einde onderzoek is leeg), wordt inOnderzoek gevuld voor betreffende verblijfstitel.
 
-  Als wel het begin van de periode (datumvan) wordt opgegeven, maar geen einde van de periode (datumtotenmet), dan worden alle verblijfstitels vanaf de datumvan in het antwoord opgenomen.
-  Als wel het einde van de periode (datumtotenmet) wordt opgegeven, maar geen begin van de periode (datumtotenmet), dan worden alle verblijfstitels tot en met de datumtotenmet in het antwoord opgenomen.
+  Als wel het begin van de periode (datumVan) wordt opgegeven, maar geen einde van de periode (datumTotEnMet), dan worden alle verblijfstitels vanaf de datumVan in het antwoord opgenomen.
```

Comment:
In regel 11 staat:

> Als de persoon al voor de periode een verblijfstitel had die binnen de gevraagde periode nog steeds geldig was, dan wordt de verblijfstitel opgenomen in het antwoord.

moet dit niet zijn:

> Als de persoon al voor de periode een verblijfstitel had die binnen **(een deel van)** de gevraagde periode nog steeds geldig was, dan wordt de verblijfstitel opgenomen in het antwoord.

Reply:
Aangepast